apps/challenges/challenge_config_utils.py

```diff
@@ -400,11 +438,19 @@ def validate_challenge_config_util(
                     )
                     error_messages.append(message)
                 else:
+                    # Addition
+                    if existing_leaderboard_config_ids and int(data["id"]) not in existing_leaderboard_config_ids:
+                        error_messages.append("ERROR: The leaderboard {} was not present in the existing leaderboard IDs. It is not possible to add new leaderboards once challenge is created. Please only use existing leaderboard IDs.".format(data["id"]))
```

Comment:
is flake8 not running on this? This single line is too long it seems.

Change message to `Leaderboard {} doesn't exist. Addition of new leaderboard after challenge creation is not allowed`